lib/tasks/eslint.rake

```diff
@@ -0,0 +1,27 @@
+task "eslint" => "eslint:check"
+
+namespace "eslint" do
+  def yarn_path
+    Rails.root.join("bin", "yarn").to_s
+  end
+
+  def config_file
+    Rails.root.join("config", "eslint.json").to_s
+  end
+
+  def js_files
+    require File.dirname(__FILE__) + "/../../config/environment"
```

Comment:
Instead of manually requiring the environment, rake tasks can be marked as depending on the rails environment e.g.

`task :check => :environment do`